CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift

```diff
@@ -35,12 +35,10 @@ public let TYPE_NODES: [Node] = [
   ),
 
   Node(
-    kind: .attributedType,
-    base: .type,
-    nameForDiagnostics: "type",
-    traits: [
-      "WithAttributes"
-    ],
+    kind: .typeSpecifier,
```

Comment:
I was going to mention alphabetical order, but seems it isn't currently ordered (though ... almost is).

Reply:
Oh, looks like we did indeed sort them in alphabetical order at some point. I just assumed that they were still in some kind of logical-ish order. I’ll sort them.